builder/replica_pipeline.cc

```diff
@@ -413,7 +431,7 @@ class pipeline_replicator : public expr_visitor {
   std::string print_expr_maybe_inlined(const expr& e) {
     if (e.defined()) {
       name_ = "$$INVALID$$";
-      e.accept(this);
+      simplify(e).accept(this);
```

Comment:
Do you need this? Given the nature of this as a debugging tool, I think we should avoid unnecessary modifications to the pipeline